src/the_way/snippet.rs

```diff
@@ -101,18 +101,28 @@ impl Snippet {
             .and_hms(0, 0, 0),
             None => Utc::now(),
         };
-        let show_default = old_code
-            .map(|c| c.split('\n').nth(1).is_none())
-            .unwrap_or(false);
-        let mut code = utils::user_input(
-            "Code snippet (<RET> to edit in external editor)",
-            if show_default { old_code } else { None },
-            show_default,
-            true,
-        )?;
-        if code.is_empty() {
-            code = utils::external_editor_input(old_code.as_deref(), &extension)?;
-        }
+
+        let code = match old_code {
+            Some(old) => {
+                if utils::confirm("Edit snippet? [y/N]", false)? {
```

Comment:
Wasn't sure if this should default to true or false

Reply:
I would say the default can be false, open for changing this if anyone requests this in the future.